--- Cement-0.8/setup.py.bak	2010-03-24 17:46:37.000000000 -0500
+++ Cement-0.8/setup.py	2010-04-30 17:44:06.930913412 -0500
@@ -91,18 +91,8 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        "ConfigObj",
-        # Only required if you want to use paster
-        "PasteScript", 
-        "tempita",
-        "jsonpickle",
-        # Required for documentation
-        # "Sphinx",
-        # "Pygments",
         ],
     setup_requires=[
-        "ConfigObj",
-        "PasteScript >= 1.7",
         ],
     entry_points="""
         [paste.global_paster_command]
